README.md

```diff
@@ -168,6 +168,7 @@ Running the game over internet currently requires setting up a relay.
 * Character movements - [Assets/Scripts/Gameplay/GameplayObjects/Character/ServerCharacterMovement.cs](Assets/Scripts/Gameplay/GameplayObjects/Character/ServerCharacterMovement.cs)
 * Client driven movements - Boss Room is server driven with anticipation animation. See [Client Driven bitesize](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/tree/main/Basic/ClientDriven) for client driven gameplay
 * Player spawn - SpawnPlayer() in [Assets/Scripts/Gameplay/GameState/ServerBossRoomState.cs](Assets/Scripts/Gameplay/GameState/ServerBossRoomState.cs)
+* Player camera setup (with cinemachine) - OnNetworkSpawn() in [Assets/Scripts/Gameplay/GameplayObjects/Character/ClientCharacter.cs](Assets/Scripts/Gameplay/GameplayObjects/Character/ClientCharacter.cs)
```

Comment:
Wouldn't it be better to add a permalink to the line where the method is implemented?

![image](https://user-images.githubusercontent.com/27730963/207093854-1ae65fb8-c5bb-4454-b43d-5bd78d82b52c.png)

Reply:
same answer https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/792#discussion_r1046229331